File: wear/src/main/java/info/nightscout/androidaps/complications/WallpaperLightComplication.java
package info.nightscout.androidaps.complications;

/*
 * Created by dlvoy on 2019-11-12
 */
public class WallpaperLightComplication extends WallpaperComplication {

    @Override
    public String getWallpaperAssetsFileName() {
        return "watch_light.jpg";
    }

    @Override
    public String getProviderCanonicalName() {
        return WallpaperLightComplication.class.getCanonicalName();
    }

    @Override
    public ComplicationAction getComplicationAction() {
        return ComplicationAction.NONE;
    };
}

File: wear/src/main/kotlin/app/aaps/wear/complications/WallpaperLightComplication.kt
package app.aaps.wear.complications

/**
 * Light Wallpaper Complication
 *
 * Displays light-themed wallpaper image
 * Asset: watch_light.jpg
 *
 */
class WallpaperLightComplication : WallpaperComplication() {

    override val wallpaperAssetsFileName: String = "watch_light.jpg"
    override fun getProviderCanonicalName(): String = WallpaperLightComplication::class.java.canonicalName!!
}